snapper/api_views.py

```diff
@@ -30,7 +30,7 @@ def post(self) -> Response:
                 jsonify({"message": "'urls' not specified"}), 400)
 
         new_task = self.create_task(request.json.get("urls"))
-        new_task.run(num_workers=app.config["num_workers"])
+        new_task.run()
```

Comment:
I thought you should be passing the value from CLI arguments?

Reply:
i only need num_workers once in scheduler.py and i cant access it from the task anyway

Reply:
Why do you think you can't? It works with the line you removed.